indexer/indexer.go

```diff
@@ -121,15 +151,15 @@ func defaultBadgerOptions(
 	opts.Compression = options.None
 
 	// Load tables into memory and memory map value logs.
-	opts.TableLoadingMode = options.MemoryMap
-	opts.ValueLogLoadingMode = options.MemoryMap
+	opts.TableLoadingMode = options.FileIO
+	opts.ValueLogLoadingMode = options.FileIO
```

Comment:
Has something changed here to warrant a change of the comment? "FileIO" seems different and more also more vague than "MemoryMap"

Reply:
This is a very good question @voyenavion , this is something I wanted @cronicc 's opinion on

Reply:
I don't know about the reasoning for switching to FileIO here, it should be slower than MemoryMap, but have less memory requirements. Maybe there was a reason for the change provided in the upstream commit message?